### PR TITLE
add Findingaids::Ead::Indexer#reindex_changed_since_days_ago

### PR DESCRIPTION
**Pull request by jgpawletko**
## Overview
* correct minor typo in comment
* add specs and implmentation for `Findingaids::Ead::Indexer#reindex_changed_since_x_days_ago`
* add `Rake` task `findingaids:ead:reindex_changed_since_x_days_ago`

### CODE REVIEW

**Comment by jgpawletko** [comment]
@barnabyalter 
if you have a couple of mins, please let me know if this looks OK to you...
Thanks!

File: lib/findingaids/ead/indexer.rb [hunk 1]
```diff
@@ -49,11 +49,18 @@ def reindex_changed_since_yesterday
...
-  # Reindex all files changed in the last day
+  # Reindex all files changed in the last week
   def reindex_changed_since_last_week
     reindex_changed(commits('--since=1.week'))
   end
 
+  # Reindex all files changed since x days ago
+  def reindex_changed_since_x_days_ago(x)
+    # assert that argument can be converted to an integer
+    days = Integer(x)
```
Comment (barnabyalter) [inline lib/findingaids/ead/indexer.rb]: @jgpawletko You could require an integer instead:

```
raise ArgumentError.new("Expecting #{x} to be an integer") unless x.is_a? Integer
```

Also for readability maybe keep the 'x' which is meant as the parameter out of the function name:

`reindex_changed_since_days_ago(days_ago)`

Do you want to just allow a `days_ago` param or should it also be configurable by date? In that case there could be two wrapper functions, the one above and `reindex_changed_since_date(date)` where date is a `Date` string.

**Comment by barnabyalter** [comment]
Hi @jgpawletko, see my comment above. If you don't see the need for these changes then I can merge it across. Will you create a Jenkins job for this?

**Comment by jgpawletko** [comment]
Hi @barnabyalter,

Thanks for the quick review!  

I'm not requiring an integer in case someone passes in a string that can be converted to an Integer, e.g.,  `'10'`.  

*Re: the change to a particular date*
I thought about it, and even started investigating requiring an ISO8601 date string, but then the "overkill" alarms started going off in my head.  

If we have a feature request for arbitrary dates, I can add it.

*re: Jenkins*
Yes, I plan to create a Jenkins job that can be updated as needed.

I'll make the method name change you suggest, but will leave the cast to Integer in place.
Updated PR coming soon :smile: .

Thanks again!

**Comment by jgpawletko** [comment]
Hi @barnabyalter , 
let me know if you want me to merge and deploy...

**Comment by sallyvermaaten** [comment]
Thank you @jgpawletko! And thank you @barnabyalter for the review!

**Pull request merged**